[2023-07-22T05:54:08.144+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-11T00:00:00+00:00 [queued]>
[2023-07-22T05:54:08.176+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-11T00:00:00+00:00 [queued]>
[2023-07-22T05:54:08.182+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:08.182+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:54:08.182+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:08.220+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-11 00:00:00+00:00
[2023-07-22T05:54:08.234+0000] {standard_task_runner.py:55} INFO - Started process 5132 to run task
[2023-07-22T05:54:08.259+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-11T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpxh1cy4kk']
[2023-07-22T05:54:08.298+0000] {standard_task_runner.py:83} INFO - Job 35: Subtask create_table
[2023-07-22T05:54:08.761+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-11T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:54:09.462+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-11T00:00:00+00:00
[2023-07-22T05:54:09.609+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:54:09.680+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:54:09.692+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:54:09.809+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230711T000000, start_date=20230722T055408, end_date=20230722T055409
[2023-07-22T05:54:10.173+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:54:10.971+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-07-22T05:56:53.002+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-11T00:00:00+00:00 [queued]>
[2023-07-22T05:56:53.084+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-11T00:00:00+00:00 [queued]>
[2023-07-22T05:56:53.088+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:53.088+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:56:53.088+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:53.151+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-11 00:00:00+00:00
[2023-07-22T05:56:53.192+0000] {standard_task_runner.py:55} INFO - Started process 5441 to run task
[2023-07-22T05:56:53.237+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-11T00:00:00+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpsywguyiq']
[2023-07-22T05:56:53.253+0000] {standard_task_runner.py:83} INFO - Job 95: Subtask create_table
[2023-07-22T05:56:53.691+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-11T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:56:54.072+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-11T00:00:00+00:00
[2023-07-22T05:56:54.125+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:56:54.170+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:56:54.181+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:56:54.209+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230711T000000, start_date=20230722T055653, end_date=20230722T055654
[2023-07-22T05:56:54.404+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:56:54.535+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
